.github/workflows/old-node.js.yml

```diff
@@ -0,0 +1,30 @@
+# Special test for the oldest version of Node.js that we "support"
+# even though the linter won't actually run. Test that the command
+# line program exits cleanly.
+
+name: Old Node CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [0.10.48]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
```

Comment:
Same thing, we could cache dependencies.

Also in my opinion, it is not necessary to test if we don't support a version, since we don't support, if you know what I mean, the user should see what version it supports, and install the correct one, so maybe we don't need to have this extra CI check, so we could have faster CI, maybe I'm wrong, just my thought.

Reply:
The `0.10.48` CLI output test is really fast it's done in about a minute.
The full-fat Node 10, 12 and 14 tests take anywhere between 7 and 10 minutes.

So we don't need to drop the old Node test for fear of having to wait around on that test. 😄